tests/ldd/enc/dep_1a/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ target_compile_options(
     # too conservative. Of the two efficient static thread-local
     # storage models, inital-exec and local-exec, we choose the most
     # optimal one.
-    -ftls-model=local-exec
+    -ftls-model=initial-exec
```

Comment:
Could you update the comment here with the one you put in the PR, here and in dep_2a/CMakeLists.txt?

```c
# The secondary modules are compiles with -ftls-model=initial-exec flag.
# This informs the compiler that the modules will be loaded along with the main module,
# rather than being dynamically loaded via dlopen. In this case, the compiler
will generate R_X86_64_TPOFF relocations for thread-local variables.
```

Reply:
Done.